doc/source/configuration/config-input.rst

```diff
@@ -0,0 +1,144 @@
+* Y: EVPN Spine POD in AZ 1,2,3... etc.
+* ZZ: Two digit uniq leaf pair identifier 01-99
+* n: a=fist leaf in pair, b=second leaf in pair
+
+Netbox Query:
+::
+    Role: EVPN Leaf
+    Name: .*(\d\d\d\d)([a-b]).* #\1=leaf_id \2=leaf_a_b
+
+CCloud Pod leafs
+################
+All CCloud pod types will be tagged to the leaf to identify sub-role:
+
+Netbox Query:
+::
+    Role: EVPN Leaf
+    Name: .*(\d\d\d\d)([a-b]).* #\1=leaf_id \2=leaf_a_b
+    Tag: CC-APOD|CC-BPOD|CC-NETPOD|CC-STPOD|CC-VPOD
+
+CND EVPN leaf types:
+####################
+The driver needs to identify certain non-pod leaf pairs to 
+push tenant configuration:
+
+ACI transit leaf, required to extend tenant networks to leagcy ACI fabric
+ 
+Netbox Query:
+::
+    Role: EVPN Leaf
+    Name: .*(\d\d\d\d)([a-b]).* #\1=leaf_id \2=leaf_a_b
+    Tag: CND-EVPN-ACI-TRANSIT
+
+EVPN Border Leaf, required to set summarization/aggregation for tenant subnet pools.
+
+Netbox Query:
+::
+    Role: EVPN Leaf
+    Name: .*(\d\d\d\d)([a-b]).* #\1=leaf_id \2=leaf_a_b
+    Tag: CND-EVPN-BORDER-LEAF
+
+*******************
+L2/L3 Networks
+*******************
+
+Tenant Network VLAN range
+##########################
+The VLAN range segments should be allocated from is per convention defined as
+::
+    2000-3750
+
+If for a device a different (or reduced) range is in effect it must be expressed in the Netbox device config context for that device
+.. code-block::
+    {
+    "cc": {
+        "net": {
+            "evpn": {
+                "tenant-vlan-range": [
+                    "2000",
+                    "2100-3000"
+                ],
+            }
+        }
+    }
+
+Infra Network DHCP Relay
+##################################
+For infra networks requiring a DHCP relay one or more Netbox Tags 
+must be added to the vlan object, one for each dhcp relay server
+in the form:
+::
+    CC-NET-EVPN-DHCP-RELAY:10.10.10.10
+    CC-NET-EVPN-DHCP-RELAY:10.11.11.11
+
+L2 Networks VLAN to VNI mapping
+#################################
+Netbox does not yet support a moddel for overlay network VNI's, the follwoing conventions are used
+
+* **Infra Regional**: VLAN X uses VNI X (VLAN 100 -> VNI 100)
+* **Infra AZ-wide**: VLAN X uses VNI [AZ-Prefix]X (VLAN 800, AZ=a -> 10800, VLAN 800, AZ=b -> 20800)
+* **Infra Pod-wide**: VLAN X re-used in many pods as local vlan 100 -> **TBD**
+* **Tenant**: CCloud platform driver should use range 1.000.000 - 1.100.000
+
+
+*******************
+Ports 
+*******************
+the driver is responsible for front ports on pod equipment, some port types require 
+certain infra VLAN's to be provisioned as well as ports beeing assembled into port-channels
+based on current port function
+
+Port infra VLANs
+###################
+Infra VLAN's required on ports are recorded on the netbox port they are reuqired on,
+for port-channels the reuqired vlans do only need to be provided on the LAG interface,
+VLAN's defined on member interfaces will be ignored for port-channel members:
```

Comment:
Well, they will only be ignored as long as the port-channel is assembled. When the PC breaks up (e.g. on DHCP/PXE install or in the baremetal usecase where user breaks their config) these VLANs probably need to be configured on the child interfaces as well.